llvm/lib/Target/X86/X86InstrShiftRotate.td

```diff
@@ -879,6 +879,26 @@ let Predicates = [HasBMI2, HasEGPR, In64BitMode] in {
   defm SHLX64 : bmi_shift<"shlx{q}", GR64, i64mem, "_EVEX">, T8, PD, REX_W, EVEX;
 }
 
+
+def immle16_8 : ImmLeaf<i8, [{
```

Comment:
maybe `immle16minus8` to be more clear?

Reply:
Good idea. I was trying to make something that took two parameters (e.g. `immle<16, 8>`) but I couldn't quite figure that out